data/pigui/modules/flight-ui/reticule.lua

```diff
@@ -420,23 +461,289 @@ local function displayManeuverData(radius)
+	["CONTROL_FIXHEADING_RADIALLY_OUTWARD"] = { icon = icons.radial_out_thin, tooltip = lui.HUD_BUTTON_FIX_RADIAL_OUT },
+	["CONTROL_FIXHEADING_KILLROT"] = { icon = icons.rotation_damping_on_thin , tooltip = lui.HUD_BUTTON_KILL_ROTATION }
+}
+
+local radial_menu_actions_orbital = {
+	{icon = ui.theme.icons.normal_thin,
+	 tooltip=lc.HEADING_LOCK_NORMAL,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_NORMAL") end},
+	{icon = ui.theme.icons.radial_out_thin,
+	 tooltip=lc.HEADING_LOCK_RADIALLY_OUTWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_RADIALLY_OUTWARD") end},
+	{icon = ui.theme.icons.retrograde_thin,
+	 tooltip=lc.HEADING_LOCK_BACKWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_BACKWARD") end},
+	{icon = ui.theme.icons.antinormal_thin,
+	 tooltip=lc.HEADING_LOCK_ANTINORMAL,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_ANTINORMAL") end},
+	{icon = ui.theme.icons.radial_in_thin,
+	 tooltip=lc.HEADING_LOCK_RADIALLY_INWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_RADIALLY_INWARD") end},
+	{icon = ui.theme.icons.prograde_thin,
+	 tooltip=lc.HEADING_LOCK_FORWARD,
+	 action=function(_) Game.player:SetFlightControlState("CONTROL_FIXHEADING_FORWARD") end},
+}
+
+local function flightAssistButton(pos)
+	local icon_size = flightAssistIconSize
+	local icon_padding = flightAssistButtonPadding
+	local button_size = flightAssistButtonSize
+	local tooltip = ""
+	local back_color = colors.lightBlueBackground:opacity(0.3)
+	-- info:
+	local flightstate = player:GetFlightState()
+	local flightcontrolstate = player:GetFlightControlState()
+	local followtarget = player:GetFollowTarget()
+	local navtarget = player:GetNavTarget()
+	local cmbtarget = player:GetCombatTarget()
+	local frmtarget = player.frameBody
+	local followmode = player:GetFollowMode()
+	local cruisedir = player:GetCruiseDirection()
+
+	local main_icon = nil -- must be overriden
+	if flightcontrolstate == "CONTROL_AUTOPILOT" then
+		local cmd = player:GetCurrentAICommand()
+		local ci = aicommand_info[cmd]
+		main_icon = icons.backward
+		tooltip = "UNKNOWN AI COMMAND?"
+		if ci then
+			main_icon = ci.icon
+			tooltip = ci.tooltip
+		end
+	elseif flightcontrolstate == "CONTROL_FIXSPEED" then
+		main_icon = cruisedir == "CRUISE_FWD" and icons.cruise_fwd or icons.cruise_up
+		tooltip = cruisedir == "CRUISE_FWD" and lui.HUD_BUTTON_CRUISE_FORWARD or lui.HUD_BUTTON_CRUISE_UP
+	else
+		main_icon = flightstate_info[flightcontrolstate].icon
+		tooltip = flightstate_info[flightcontrolstate].tooltip
+	end
+
+	-- background
+	ui.addIcon(pos, icons.follow_fill, back_color, Vector2(button_size, button_size), ui.anchor.center, ui.anchor.center, tooltip)
+	ui.addIcon(pos, icons.follow_edge, back_color, Vector2(button_size, button_size), ui.anchor.center, ui.anchor.center, tooltip)
```

Comment:
Is there a reason this is drawn using multiple icons composed on top of each other instead of using `ui.addCircle` + `ui.addCircleFilled`? (Or even just a single icon?) That would save two slots in the icon atlas, at the expense of generating some extra vertex data. It's not a required optimization by any means, just a bit weird seeing it here.

Reply:
![image](https://user-images.githubusercontent.com/18342621/169668265-7ef1868d-ee21-4526-aac0-9fb5ca13191c.png)
The fact is that in the follow mode even 3 icons are superimposed there, and I would not want to adjust the diameters and thicknesses of the circles, all this is provided in inkscape.